src/sources/tripadvisor/pois/convert.rs

```diff
@@ -145,9 +146,27 @@ pub fn build_poi(property: Property, geofinder: &AdminGeoFinder) -> Result<(u32,
         name: poi_type_name,
     };
 
+    let opening_hours = property
+        .hours
+        .inner
+        .iter()
+        .filter(|day| day.time.is_some())
+        .map(|day| {
+            let opening_times: String = day
+                .time
+                .iter()
+                .flatten()
+                .map(|time| format!("{}-{}", time.open_time, time.close_time))
+                .join(",");
+            format!("{} {}", day.name.get(0..2).unwrap_or(""), opening_times)
+        })
```

Comment:
I'd argue that this would be more readable with a `filter_map` but that's purely cosmetic (this never empty `flatten` is slightly hard to understand).